lib/beaker-pe/install/pe_utils.rb

```diff
@@ -1087,6 +1096,9 @@ def register_feature_flags!(opts)
         # @param [Hash] opts The Beaker options hash
         # @return [BeakerAnswers::Answers] the generated answers object
         def generate_installer_conf_file_for(host, hosts, opts)
+          possible_mco_enabled_setting = get_mco_setting(hosts)
+          opts ||= {}
+          opts.merge!(possible_mco_enabled_setting)
```

Comment:
I think the problem here is that what generate_installer_conf_file_for receives is the Beaker#options.
If there is already an options[:answers] then this will overwrite it rather than merge into it.

Reply:
Agreed, this should probably be something like `opts = possible_mco_enabled_setting.merge(opts)`, so that the mco setting is a default and explicit opts will override it.

Reply:
I think we still need to deal with ensuring that the :answers hash of opts gets merged rather than knocked out.

For that, we could require 'deep_merge' and use that; it's a vendored dependency of puppet.

Reply:
Thanks, good catch.